assets/css/main.css

```diff
@@ -1,3 +1,21 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
```

Comment:
would this also work in our `./src/renderer/App.css` ?

Reply:
Oh, yeah, totally. That's where it should go actually.

Reply:
I'm out of time for this today, feel free to make that change directly. Or I'll do next week.